### ISSUE TITLE
Locale issue

### ISSUE THREAD

**Issue by ruienf** [ulsdevteam/ojs-plum-plugin#10]
HiI

I'm using OJS 3.3.04 and the plugin has a locale issue (it's showing only the message keys like ##plugins.generic.forthcoming.displayName##).

I think it's because it still uses the locale.xml file.

I've tried to change it lo locale.po and it's working fine now.

I send you the two files I've made (they need to be polished...) using en_US and pt_PT languages.

Rui Pedro Silva
[locale en_US pt_PT.zip](https://github.com/ulsdevteam/ojs-plum-plugin/files/6404272/locale.en_US.pt_PT.zip)

**Comment by ruienf** [comment]
The previous files had quotation marks issues.

Corrected files (added pt_BR, which is a duplicate of pt_PT):

[Locale PT BR US.zip](https://github.com/ulsdevteam/ojs-plum-plugin/files/6404469/Locale.PT.BR.US.zip)

**Comment by ctgraham** [comment]
Thanks, @ruienf .

You might be interested to know about the `XmlToPo.php` tool in `lib/pkp/tools/` of OJS 3.2, which can help to automatically convert legacy XML files to .po files.

I've fixed the malformed XML in pt_BR and I have run the tool on the existing XML files.

I then copied some of your cleanup of the whitespace in pt_BR.  I did not copy your removal of the HTML from en_US and pt_BR.  Does that HTML function differently in 3.3?

Please take a look at these changes.  If they are acceptable, I will copy pt_BR to pt_PT and I will release a new version of the plugin.  If you would like to make additional changes to improve the translations, I will wait for those to release the new version.

**Comment by ruienf** [comment]
Hi @ctgraham !

Thank you for your reply; I didn't know about XmlToPo.php, great tip! 

I only removed HTML because I didn't know if it would work with .po files (I'm only a beginner at OJS...).

I reviewed the pt_BR locale.po and I made minor changes for better language accuracy; The same file can be used in pt_PT locale.

I set it as .txt to upload it here:

[locale.txt](https://github.com/ulsdevteam/ojs-plum-plugin/files/6414191/locale.txt)

Thank you for the opportunity of helping and increasing my OJS knowledge!

Kind regards,

Rui Pedro Silva
